catalog/core/catalog-core-impl/metacard-type-registry/src/test/java/ddf/catalog/extensiblemetacard/MetacardTypeRegistryTest.java

```diff
@@ -447,6 +380,70 @@ public void testUnregister()
         mtr.unregister(sampleMetacardTypeB());
     }
 
+    @Test
+    public void testAddRemoveMetacardType() throws MetacardTypeUnregistrationException {
+        final String id = "ID-123";
+        final String schema = "http://example.com/namespace";
+        final String typeName = "myTypeName";
+
+        MetacardTypeRegistryImpl metacardTypeRegistry = new MetacardTypeRegistryImpl() {
+            @Override
+            protected BundleContext getBundleContext() {
+                return mockBundleContext;
+            }
+        };
+        ServiceReference<MetacardType> svcRef = mock(ServiceReference.class);
+
+        // Test blank typeName
```

Comment:
Seems like you're testing more than one thing in `testAddRemoveMetacardType()` - each should have its own test method.